packages/sdk/src/sdk/sdk.withdraw.ts

```diff
@@ -94,8 +94,14 @@ export async function getWithdrawData<
     destinationPolkadot.getGenericBalance(destinationAccount, config.balance),
     destinationPolkadot.getExistentialDeposit(),
   ]);
-  const destinationFee = BigInt(config.weight * config.feePerWeight);
-  const min = destinationFee + existentialDeposit;
+
+  const assetMinBalance = config.sourceMinBalance
```

Comment:
You should include it in Promise.all to reduce the time that the user will need to wait.

Reply:
Sorry, I missed that, it is done now